components/scanners/zaproxy/overwrite-metadata/cmd/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/go-errors/errors"
+	"google.golang.org/protobuf/encoding/protojson"
+
+	"github.com/smithy-security/smithy/sdk/component"
+	ocsffindinginfo "github.com/smithy-security/smithy/sdk/gen/ocsf_ext/finding_info/v1"
+)
+
+func main() {
```

Comment:
Any reason why this is not using the SDK?

Reply:
can a component have two pieces? this is not necessarily a component, instead this is just a quick hack to propagate metadata for a workflow that doesn't fit squarely into the smithy mold

Reply:
I think it's an ok solution, the proper long term solution involves a more flexible execution engine and a different type of event as an entrypoint for the state machine